src/routes/workflow.js

```diff
@@ -147,6 +147,58 @@ function init (server, { middlewares, helpers } = {}) {
     const result = await requester.send(params)
     return result
   }))
+
+  // //////////// //
+  // WORKFLOW LOG //
+  // //////////// //
+
+  server.get({
+    name: 'workflow.listLogs',
+    path: '/workflow-logs'
+  }, checkPermissions([
+    'workflowLog:list:all'
+  ]), wrapAction(async (req, res) => {
+    const fields = [
+      'orderBy',
+      'order',
+      'page',
+      'nbResultsPerPage',
+
+      'id',
+      'createdDate',
+      'workflowId',
+      'eventId',
+      'runId',
+      'statusCode',
+    ]
+
+    const payload = _.pick(req.query, fields)
+
+    let params = populateRequesterParams(req)({
+      type: 'listLogs',
+      workflowType: req.query.type,
```

Comment:
`workflowType` is confusing. This is rather `logType`.

Reply:
Indeed it's confusing. I updated the variable name.